dockerfile/ubuntu-20.04-amd64.Dockerfile

```diff
@@ -33,15 +33,16 @@ ENV PYTHON_ENV_DIR=${TT_METAL_INFRA_DIR}/tt-metal/python_env
 # ENV PATH="$PYTHON_ENV_DIR/bin:$PATH"
 
 # Copy requirements from tt-metal folders with requirements.txt docs
-COPY /docs/requirements-docs.txt ${TT_METAL_INFRA_DIR}/tt-metal/docs/.
+COPY /docs/requirements-docs.txt ${TT_METAL_INFRA_DIR}/tt-metal/docs/requirements-docs.txt
```

Comment:
Is everything here necessary for running of Python tests? This is related to the effort of creation of Release image - create a separate ticket to track that work.

Reply:
Create an issue

Reply:
https://github.com/tenstorrent/tt-metal/issues/11974

Reply:
Depends on if you run the docs pipeline in docker as well , in which case yes, but we don't have to.